CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+## 1.6.0 / 2017-04-14
+
+* [CHANGE] Replaced the **experimental** remote write implementations for
+  various backends by a generic write interface with example bridge
+  implementation for various backends.
+* [FEATURE] New flag `-storage.local.target-heap-size` to tell Prometheus about
+  the desired heap size. This deprecates the flags
+  `-storage.local.memory-chunks` and `-storage.local.max-chunks-to-persist`,
+  which are kept for backward compatibility.
+* [FEATURE] Add `check-metrics` to `promtool` to lint metric names.
+* [FEATURE] Auto-tweak times between checkpoints to limit time spent in
+  checkpointing to 50%.
+* [FEATURE] Add Joyent Triton discovery.
+* [FEATURE] `Scrape-Timeout-Seconds` header in HTTP scrape requests.
+* [FEATURE] Remote read interface, including example for InfluxDB. **Experimental.**
+* [FEATURE] Enable Consul SD to connect via TLS.
+* [FEATURE] Marathon SD supports multiple ports.
+* [FEATURE] Marathon SD supports bearer token for authentication.
+* [FEATURE] Match addresses with or without declared ports in Kubernetes SD.
+* [FEATURE] Custom timeout for queries.
+* [FEATURE] Expose `buildQueryUrl` in console templates.
+* [FEATURE] Add `rickshawGraph` property to the graph object in console
+  templates.
+* [FEATURE] New metrics exported by Prometheus itself:
+  * Summary `prometheus_engine_query_duration_seconds`
+  * Counter `prometheus_evaluator_iterations_missed_total`
+  * Counter `prometheus_evaluator_iterations_total`
+  * Gauge `prometheus_local_storage_open_head_chunks`
+  * Gague `prometheus_local_storage_target_heap_size`
+* [ENHANCEMENT] Reduce shut-down time by interrupting an ongoing checkpoint
+  before starting the final checkpoint.
+* [ENHANCEMENT] Improved crash recovery deals better with certain index
+  corruptions.
+* [ENHANCEMENT] Graphing deals better with constant time series.
+* [ENHANCEMENT] Retry remote writes on recoverable errors.
+* [ENHANCEMENT] Evict unused chunk descriptors during crash recovery to limit
+  memory usage.
+* [ENHANCEMENT] Smoother disk usage during series maintenance.
+* [ENHANCEMENT] Targets on targets page sorted by instance within a job.
+* [ENHANCEMENT] Sort labels in federation.
+* [ENHANCEMENT] Set `GOGC=40` by default, which results in much better memory
+  utilization at the price of slightly higher CPU usage. If `GOGC` is set by
+  the user, it is still honored as usual.
+* [ENHANCEMENT] Close head chunks after being idle for the duration of the
+  configured staleness delta. This helps to persist and evict head chunk of
+  stale series more quickly.
+* [ENHANCEMENT] Stricter checking of relabel config.
+* [ENHANCEMENT] Cach busters for static web content.
+* [ENHANCEMENT] Send user-agent header during scrapes.
```

Comment:
We always sent one, might be better to include something like `Prometheus specific`?